### DISCUSSION TITLE
OCR bill

### DISCUSSION THREAD

**Discussion by TranMinh11** [PaddlePaddle/PaddleOCR#13063] [Ideas & Features]
Currently, I am learning about OCR. I have a problem about invoices that require OCR to extract tables from a PDF file. But when I use PaddleOCR, there are many text parts that are wrong or missing and the table structure is not like the original file. Please give me some ideas. Thanks

**Comment by GreatV** [comment]
Could you provide some images so we can test?

**Reply by TranMinh11** [reply]
I sent an email to you. Please check for me. Thanks

**Reply by GreatV** [reply]
@TrM11 got it.

**Reply by GreatV** [reply]
@TrM11,I think you could try installing the main branch, the results don't look bad.

**Reply by TranMinh11** [reply]
With the main.png file I get relatively good results but the main_ocr.png file is not like that..I have sent the results via email to you. Please check for me
